### PR TITLE
Some fixes to make it compatible with the latest Xenko version

### PR DESCRIPTION
**Pull request by stefnotch**
I left a bunch of comments that explain my changes. 

Anyways, this is a pretty cool project. I'm so glad that this exists.

### CODE REVIEW

File: XenkoShaderExplorer/MainViewModel.cs [hunk 1]
```diff
@@ -64,7 +65,16 @@ public MainViewModel()
         {
             try
             {
-                RootShaders = BuildShaderTree().ToList();
+				_path = Directory
```
Comment (stefnotch) [inline XenkoShaderExplorer/MainViewModel.cs]: It automatically figures out the correct path. (It takes the first path that starts with the word "Xenko")

File: XenkoShaderExplorer/MainViewModel.cs [hunk 2]
```diff
@@ -64,7 +65,16 @@ public MainViewModel()
...
+					.EnumerateDirectories(BASE_PATH)
+					.First(s => 
+						System.IO.Path
+							.GetFileName(s)
+							.ToUpper()
+							.StartsWith("XENKO")
+					);
```
Comment (stefnotch) [inline XenkoShaderExplorer/MainViewModel.cs]: git@github.com:stefnotch/Shader-Explorer-For-Xenko.git

Comment (stefnotch) [inline XenkoShaderExplorer/MainViewModel.cs:75]: It automatically figures out the correct path.
(It takes the first path whose folder name starts with "Xenko")

File: XenkoShaderExplorer/MainViewModel.cs [hunk 3]
```diff
@@ -116,8 +126,13 @@ private IEnumerable<Shader> BuildShaderTree()
...
             foreach (var shader in shaders.Values)
             {
-                var text = File.ReadAllLines(shader.Path)
-                    .FirstOrDefault(s => s.Trim().StartsWith("shader"));
+				var stop = false;
+				var text = string.Join(" ",
+								File.ReadAllLines(shader.Path)
+									.SkipWhile(s => !s.Trim().StartsWith("shader")) //From shader
+									.TakeWhile(s => !s.Contains("{")) // To the bracket (exclusive)
+						);
```
Comment (stefnotch) [inline XenkoShaderExplorer/MainViewModel.cs:134]: So, there are shaders like this:
```
shader SOMENAME: A,
            B,
            C
    {
```
This code finds the first line that starts with shader and keeps taking the next few lines until the `{`. Then, it merges the lines into a single string.

File: XenkoShaderExplorer/MainViewModel.cs [hunk 4]
```diff
@@ -132,12 +147,16 @@ private IEnumerable<Shader> BuildShaderTree()
                         var baseShaderNames = baseShaderDeclaration
                             .Split(',')
                             .Select(s => s.Trim());
+						if (!baseShaderNames.Contains("ShadowMapCasterBase")) { //I have no clue why this shader doesn't exist. >w>
```
Comment (stefnotch) [inline XenkoShaderExplorer/MainViewModel.cs:150]: As the comment says, this shader simply doesn't exist.

**Comment by SvenEV** [comment]
Hey, it's great to see that this project is still useful for someone :) As you noticed, it is not in active development. So thank you for making it compatible with the current version of Xenko!

**Pull request merged**